dll/win32/shell32/wine/PolicyData.h

```diff
@@ -0,0 +1,237 @@
+// NOTE: REST_NOSTARTPAGE never really existed.
+DEFINE_POLICY( REST_NOAUTOTRAYNOTIFY,            "Explorer",       "NoAutoTrayNotify"                 ),
+DEFINE_POLICY( REST_NOTASKGROUPING,              "Explorer",       "NoTaskGrouping"                   ),
+DEFINE_POLICY( REST_NOCDBURNING,                 "Explorer",       "NoCDBurning"                      ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN2KSP3)
+DEFINE_POLICY( REST_MYCOMPNOPROP,                "Explorer",       "NoPropertiesMyComputer"           ),
+DEFINE_POLICY( REST_MYDOCSNOPROP,                "Explorer",       "NoPropertiesMyDocuments"          ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+DEFINE_POLICY( REST_NODISPLAYAPPEARANCEPAGE,     "System",         "NoDispAppearancePage"             ),
+DEFINE_POLICY( REST_NOTHEMESTAB,                 "Explorer",       "NoThemesTab"                      ),
+DEFINE_POLICY( REST_NOVISUALSTYLECHOICE,         "System",         "NoVisualStyleChoice"              ),
+DEFINE_POLICY( REST_NOSIZECHOICE,                "System",         "NoSizeChoice"                     ),
+DEFINE_POLICY( REST_NOCOLORCHOICE,               "System",         "NoColorChoice"                    ),
+DEFINE_POLICY( REST_SETVISUALSTYLE,              "System",         "SetVisualStyle"                   ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN2KSP3)
+DEFINE_POLICY( REST_STARTRUNNOHOMEPATH,          "Explorer",       "StartRunNoHOMEPATH"               ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+DEFINE_POLICY( REST_NOSTARTPANEL,                "Explorer",       "NoSimpleStartMenu"                ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+DEFINE_POLICY( REST_NOUSERNAMEINSTARTPANEL,      "Explorer",       "NoUserNameInStartMenu"            ),
+DEFINE_POLICY( REST_NOMYCOMPUTERICON,            "NonEnum", "{20D04FE0-3AEA-1069-A2D8-08002B30309D}"  ),
+DEFINE_POLICY( REST_NOSMNETWORKPLACES,           "Explorer",       "NoStartMenuNetworkPlaces"         ),
+DEFINE_POLICY( REST_NOSMPINNEDLIST,              "Explorer",       "NoStartMenuPinnedList"            ),
+DEFINE_POLICY( REST_NOSMMYMUSIC,                 "Explorer",       "NoStartMenuMyMusic"               ),
+DEFINE_POLICY( REST_NOSMEJECTPC,                 "Explorer",       "NoStartMenuEjectPC"               ),
+DEFINE_POLICY( REST_NOSMMOREPROGRAMS,            "Explorer",       "NoStartMenuMorePrograms"          ),
+DEFINE_POLICY( REST_NOSMMFUPROGRAMS,             "Explorer",       "NoStartMenuMFUprogramsList"       ),
+#endif
+#if (NTDDI_VERSION >= NTDDI_WINXP)
+DEFINE_POLICY( REST_HIDECLOCK,                   "Explorer",       "HideClock"                        ),
+DEFINE_POLICY( REST_NOLOWDISKSPACECHECKS,        "Explorer",       "NoLowDiskSpaceChecks"             ),
+DEFINE_POLICY( REST_NODESKTOPCLEANUP,            "Explorer",       "NoDesktopCleanupWizard"           ),
+#endif
+DEFINE_POLICY( REST_NOENTIRENETWORK,             "Network",        "NoEntireNetwork"                  ),
```

Comment:
Could you verify that these two: NOENTIRENETWORK and NODESKTOPCLEANUP, are absent in shell32 from Vista and later? (i.e. they are in the table for NTDDI_VERSION < NTDDI_LONGHORN).
See also https://www.geoffchappell.com/studies/windows/shell/shell32/api/util/restrictions.htm

Reply:
In Win10, two strings `L"NoDesktopCleanupWizard"` and `L"NoEntireNetwork"` not found.